src/modules/flow/twitter/twitter.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "twitter",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD-3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "name": "twitter/client",
+      "category": "output/network",
+      "description": "A twitter's node client",
+      "methods": {
+        "close": "twitter_close",
+        "open": "twitter_open"
+      },
+      "options": {
+	"members": [
+	  {
+	    "data_type": "string",
+	    "description": "The consumer key (application's token)",
+	    "name": "consumer_key"
+	  },
+	  {
+	    "data_type": "string",
+	    "description": "The consumer secret key (application's secret token)",
+	    "name": "consumer_secret"
+	  }
+	],
+	"version": 1
+      },
+      "in_ports": [
+        {
+          "data_type": "string",
+          "description": "The token returned on twitter's oauth process",
+          "methods": {
+            "process": "token_process"
+          },
+          "name": "TOKEN"
+        },
+	{
+          "data_type": "string",
+          "description": "Post the string that comes into twitter",
+          "methods": {
+            "process": "post_process"
+          },
+          "name": "TWEET"
```

Comment:
nope, i thought we were asking to create several different nodes with specific functionalities, for example:

``twitter/tweet`` -> just post a tweet (with its possible options)
``twitter/timeline`` -> just deal with timeline
``twitter/account`` -> change photo, personal information, and so on

Of course, we can have a generic version that receives a json, but i guess those small nodes will be easier to use on a flow